src/tests/Regressions/coreclr/GitHub_45929/test45929.cs

```diff
@@ -0,0 +1,124 @@
+using System;
```

Comment:
Nit: License header

Reply:
Oops, all of my recent regression tests are missing licence header. I'll fix them.